### PR TITLE
Task #283 add hide show expired switch

### PR DESCRIPTION
**Pull request by NemTechCN**
solve task 283 and update mosaic/namespace list stulesheet

### CODE REVIEW

**Comment by decentraliser** [comment]
wait I'm going to make some changes to this PR and merge it

**Comment by decentraliser** [comment]
will merge it after the "F" PR merge

notice that I had already done the new FormattedTransactions

File: src/core/utils/mosaics.ts [hunk 1]
```diff
@@ -30,11 +31,40 @@ export const renderMosaics = (
...
+    //
+    // if (!items.length) return 'Loading...'
+    //
+    // const networkMosaicIndex = items.findIndex(({name}) => name === store.state.account.networkCurrency.hex)
+    // if (networkMosaicIndex <= 0) {
+    //     return items.map(({name, amount}) => `${amount} [${name}]`).join(', ')
+    // }
+    // const networkCurrency = items.splice(networkMosaicIndex, 1)
+    // items.unshift(networkCurrency[0])
+    // return items.map(({name, amount}) => `${amount} [${name}]`).join(', ')
```
Comment (decentraliser) [inline src/core/utils/mosaics.ts]: delete if unused

File: src/views/wallet/wallet-switch/WalletSwitchTs.ts [hunk 2]
```diff
@@ -82,10 +82,7 @@ export class WalletSwitchTs extends Vue {
     }
 
     switchWallet(newActiveWalletAddress) {
-        this.$store.commit(
-            'SET_WALLET',
-            this.walletList.find(({address}) => address === newActiveWalletAddress)
-        )
+        AppWallet.switchWallet(newActiveWalletAddress, this.walletList, this.$store)
```
Comment (decentraliser) [inline src/views/wallet/wallet-switch/WalletSwitchTs.ts]: please revert this change
active wallet changes are (and should only be) managed in App.vue (`this.$watchAsObservable('address')`)

Comment (decentraliser) [reply src/views/wallet/wallet-switch/WalletSwitchTs.ts]: and watchAsObservable will trigger `onWalletChange`, that has the `walletSwitch` method

Comment (decentraliser) [reply src/views/wallet/wallet-switch/WalletSwitchTs.ts]: image

**Comment by NemTechCN** [comment]
> Switching fast between A and B put A's balance on B account
> 
> image

fixed

**Comment by decentraliser** [comment]
merged into develop
